configdialog/lxqtpageselectwidget.cpp

```diff
@@ -102,12 +103,14 @@ QSize PageSelectWidget::sizeHint() const
     int w = 0;
     for(int i=0; i< count(); ++i)
     {
-        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignLeft | Qt::TextWordWrap, item(i)->text());
+        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignHCenter | Qt::TextWordWrap, item(i)->text());
         w = qMax(w, rect.width());
     }
+    // consider the icon size too
+    w = qMax(w, QApplication::style()->pixelMetric(QStyle::PM_IconViewIconSize));
```

Comment:
> Please see qcommonstyle.cpp → QCommonStyle::pixelMetric → case PM_ScrollBarExtent:! widget has no part in it.

That's true, but `QCommonStyle` is the "base" style. See the `QStyleSheetStyle::pixelMetric()`...